app/constants/languages.js

```diff
@@ -1,102 +1,101 @@
 const languages = [
   { value: 'af', label: 'Afrikaans' },
   { value: 'sq', label: 'Albanian' },
-  { value: 'gsw-fr', label: 'Alsatian (France)' },
-  { value: 'am-et', label: 'Amharic (Ethiopia)' },
+  { value: 'gsw', label: 'Alsatian' },
+  { value: 'am', label: 'Amharic' },
   { value: 'ar', label: 'Arabic' },
   { value: 'hy', label: 'Armenian' },
-  { value: 'as-in', label: 'Assamese (India)' },
+  { value: 'as', label: 'Assamese' },
   { value: 'az', label: 'Azeri' },
-  { value: 'ba-ru', label: 'Bashkir (Russia)' },
+  { value: 'ba', label: 'Bashkir' },
   { value: 'eu', label: 'Basque' },
   { value: 'be', label: 'Belarusian' },
   { value: 'bn', label: 'Bengali' },
-  { value: 'bn-bd', label: 'Bengali (Bangladesh)' },
-  { value: 'bn-in', label: 'Bengali (India)' },
+  { value: 'bs', label: 'Bosnian' },
   { value: 'bs-cyrl-ba', label: 'Bosnian (Cyrillic, Bosnia and Herzegovina)' },
   { value: 'bs-latn-ba', label: 'Bosnian (Latin, Bosnia and Herzegovina)' },
-  { value: 'br-fr', label: 'Breton (France)' },
+  { value: 'br', label: 'Breton' },
   { value: 'bg', label: 'Bulgarian' },
   { value: 'ca', label: 'Catalan' },
   { value: 'zh-cn', label: 'Chinese (Simplified)' },
   { value: 'zh-tw', label: 'Chinese (Traditional)' },
   { value: 'hr', label: 'Croatian' },
   { value: 'cs', label: 'Czech' },
   { value: 'da', label: 'Danish' },
-  { value: 'prs-af', label: 'Dari (Afghanistan)' },
+  { value: 'prs', label: 'Dari' },
   { value: 'div', label: 'Divehi' },
   { value: 'nl', label: 'Dutch' },
   { value: 'en', label: 'English' },
   { value: 'en-gb', label: 'English (United Kingdom)' },
   { value: 'en-us', label: 'English (United States)' },
```

Comment:
❓ Question: in Pandora PR [848](https://github.com/zooniverse/pandora/pull/848), both 'en-gb' and 'en-us' are **removed** from the Zooniverse Translations website's list of "languages you can set translations for".

Is this an oversight, or are we maintaining 'en-gb' and 'en-us' on PFE for backwards compatibility? (See recent [Slack message:](https://zooniverse.slack.com/archives/C583QHN49/p1719423510266569?thread_ts=1719421227.260789&cid=C583QHN49) 68 projects have en-gb and 217 projects have en-us)

If we ARE making a conscious decision to only have `en-gb` and `en-us` in PFE's languages list but not Pandora's languages list, we should add a comment here saying something like `/* English dialects should exist only PFE's language list. Do not delete these entries from PFE nor add them to Pandora when syncing the language lists of both repos. */`

Reply:
It is most important to limit the language list in Pandora, so as to limit the language translations that can be created. For PFE, the list can be more inclusive, because to the best of my knowledge, it is only used to transform a language code into a language name.  

In a feat of cowardice, I opt to keep en-gb and en-us in the list because there are multiple projects and workflows that have these language codes as their primary_language value.  While these additional entries shouldn't be needed, there is little penalty for doing so.

Note: whether en-us and en-gb primary_language values should be allowed or cleaned up is a whole issue unto itself; out of scope for now.

I'm happy to add the comment if you feel strongly we should, but I feel it is not necessary.

Reply:
Thanks for the clarification, Cliff. 👍 I'm on board with the reasons you have.

I do _very, very strongly_ suggest we add the comment about the PFE code's en-gb and en-us entries, because I know that in the future, a dev (e.g. me) is gonna go _"I'll just copy-paste the languages.js file wholesale to/from pandora to sync the changes, it'll be fast & easy! What could go wrong? [A-hyuck!](https://www.youtube.com/watch?v=0kAEthfslsE)",_ and an inline comment in the code will make it very obvious to a reviewer that they shouldn't be doing that.

Reply:
Thanks for the example -- yes, I see the utility of the comment now. I'll get that in (here and on Pandora just for completeness) before merging early next week.

Reply:
From an accessibility perspective, `en-US` and `en-GB` can be read and pronounced differently by screenreaders, so specifying a dialect canbe useful for screenreaders, in cases where there are differences of spelling or pronunciation. I assume that's true for other dialects too eg. Mexican Spanish is pronounced very differently from European Spanish. One fairly common example, in English, is that VoiceOver/Safari with a UK voice will consistently mispronounce 'favorite' on Zooniverse project pages.